approaching_emergency_vehicle_plugin/test/test_approaching_erv.cpp

```diff
@@ -938,6 +938,78 @@ namespace approaching_emergency_vehicle_plugin{
+        worker_node->has_planned_upcoming_lc_ = true;
+        worker_node->upcoming_lc_params_.is_right_lane_change = true;
+
+        status_msg = worker_node->generateApproachingErvStatusMessage();
+        ASSERT_EQ(status_msg.type, carma_msgs::msg::UIInstructions::INFO);
+        ASSERT_EQ(status_msg.msg, "HAS_APPROACHING_ERV:1,TIME_UNTIL_PASSING:11.5634,EGO_VEHICLE_ACTION:Changing lanes to the right.");
+
+        // Generate status message when an ERV is being tracked in state WAITING_FOR_APPROACHING_ERV and verify the output
+        worker_node->has_tracked_erv_ = true;
+        worker_node->tracked_erv_.seconds_until_passing = 11.563429203;
+        worker_node->transition_table_.event(ApproachingEmergencyVehicleEvent::ERV_ABOUT_TO_PASS_IN_PATH);
+
+        status_msg = worker_node->generateApproachingErvStatusMessage();
+        ASSERT_EQ(status_msg.type, carma_msgs::msg::UIInstructions::INFO);
+        ASSERT_EQ(status_msg.msg, "HAS_APPROACHING_ERV:1,TIME_UNTIL_PASSING:11.5634,EGO_VEHICLE_ACTION:Remaining in the current lane at the speed limit.");
+
+        // Generate status message when an ERV is being tracked in state SLOWING_DOWN_FOR_ERV with ego vehicle travelling significantly above the target reduced speed, and verify the output
+        worker_node->transition_table_.event(ApproachingEmergencyVehicleEvent::ERV_PASSING_IN_PATH);
+
+        carma_planning_msgs::msg::ManeuverPlan maneuver_plan;
+        carma_planning_msgs::msg::Maneuver maneuver;
+        maneuver.type = carma_planning_msgs::msg::Maneuver::LANE_FOLLOWING;
+        maneuver.lane_following_maneuver.end_speed = 6.7056; // (m/s) equal to 15 mph
+        maneuver_plan.maneuvers.push_back(maneuver);
+        worker_node->latest_maneuver_plan_ = maneuver_plan;
+
+        worker_node->current_speed_ = 15.0; // (m/s), not within threshold (config_.reduced_speed_buffer) of first maneuver's target speed (end_speed)
+
+        status_msg = worker_node->generateApproachingErvStatusMessage();
+        ASSERT_EQ(status_msg.type, carma_msgs::msg::UIInstructions::INFO);
+        ASSERT_EQ(status_msg.msg, "HAS_APPROACHING_ERV:1,TIME_UNTIL_PASSING:11.5634,EGO_VEHICLE_ACTION:Remaining in the current lane and slowing down to a reduced speed limit.");
+
+        // Generate status message when an ERV is being tracked in state SLOWING_DOWN_FOR_ERV with ego vehicle travelling near the the target reduced speed, and verify the output
+        worker_node->current_speed_ = 6.8; // (m/s), within threshold (config_.reduced_speed_buffer) of first maneuver's target speed (end_speed)
+
+        status_msg = worker_node->generateApproachingErvStatusMessage();
+        ASSERT_EQ(status_msg.type, carma_msgs::msg::UIInstructions::INFO);
+        ASSERT_EQ(status_msg.msg, "HAS_APPROACHING_ERV:1,TIME_UNTIL_PASSING:11.5634,EGO_VEHICLE_ACTION:Remaining in the current lane at a reduced speed limit.");
```

Comment:
The same comments for the set reduced speed limit value

Reply:
Fixed in [99d6024](https://github.com/usdot-fhwa-stol/carma-platform/pull/2064/commits/99d6024c9e7a4bbfba067ff48a2d156f2eb14777).